charts/newrelic-infra-operator/values.yaml

```diff
@@ -23,8 +23,8 @@ image:
 
 admissionWebhooksPatchJob:
   image:
-    repository: jettech/kube-webhook-certgen
-    tag: v1.5.0
+    repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+    tag: v1.1.1
```

Comment:
Small discrepancy, above we say `v1.1.0`.

Related: Did the ingress-nginx fellas _downgrade_ the version? Seems strange but might be okay.

Reply:
it is v1.1.1 somehow I messed up copying pasting 😖 
https://github.com/kubernetes/k8s.io/blob/055cf8523bdbf31c00a47d20f057fc2c231edbb7/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L126

Reply:
Fixed it and I even remembered adding the Co-authored 🤯

Reply:
Are we deprecating any Kubernetes version with this change that we must be aware of?

Reply:
No, this image is already be used by different charts. All changes are related to the already in place <1.15 deprecation